index.js

```diff
@@ -73,37 +77,37 @@
       return tip
     }
 
-    // Public: Proxy attr calls to the d3 tip container.  Sets or gets attribute value.
+    // Public: Proxy attr calls to the d3 tip container.
+    // Sets or gets attribute value.
     //
     // n - name of the attribute
     // v - value of the attribute
     //
     // Returns tip or attribute value
-    tip.attr = function(n, v) {
+    tip.attr = function(n, _v) {
       if (arguments.length < 2 && typeof n === 'string') {
         return getNodeEl().attr(n)
-      } else {
-        var args =  Array.prototype.slice.call(arguments)
-        d3.selection.prototype.attr.apply(getNodeEl(), args)
       }
 
+      var args =  Array.prototype.slice.call(arguments)
+      d3.selection.prototype.attr.apply(getNodeEl(), args)
       return tip
     }
 
-    // Public: Proxy style calls to the d3 tip container.  Sets or gets a style value.
+    // Public: Proxy style calls to the d3 tip container.
+    // Sets or gets a style value.
     //
     // n - name of the property
     // v - value of the property
     //
     // Returns tip or style property value
-    tip.style = function(n, v) {
+    tip.style = function(n, _v) {
```

Comment:
It's not obvious what the underscore represents here.  Though this method doesn't use the `v` argument directly, it does pass it along to set the value of the given `style`.